modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -687,6 +687,10 @@ use {{invokerPackage}}\ObjectSerializer;
         }
         {{/formParams}}
 
+        if ($contentType === 'multipart/form-data') {
+            $multipart = true;
```

Comment:
What I did not manage to figure out: Is it possible to move this check to _generation time_?

Reply:
thanks for the PR

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L4610

what about using `isMultipart` in codegen operation?

Reply:
Thanks for the hint :slightly_smiling_face: